corona-warn-app/cwa-app-android
# Notifications delta onboarding (EXPOSUREAPP-9615)

## Changes
Delta onboarding for new notification management.

<img width="207" alt="Screenshot 2021-09-29 at 13 04 01" src="https://user-images.githubusercontent.com/64849422/135250640-a12ab26d-3918-477c-8ac3-2f4c717d0d98.png">


## Story
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9497
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9615

## How to test
1. Go to testers menu
2. Onboarding options
3. Set version to previous (e.g. 2110000)
4. Scroll down to Notifications and Analytics part
5. Uncheck either only notifications or both
6. The correct delta onboarding order is: New features -> Notifications -> PPA
7. Check navigation from Notifications Delta Onboarding screen to application notification settings (OS level)

Note for testers: at this point it is impossible to get in position when notifications onboarding is complete and analytics is not, as analytics onboarding were introduced earlier (about Feb 2021).
